pipelines/integration_resolver_pipeline_environment_pass.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: integration-resolver-pipeline-pass
+spec:
+  tasks:
+    - name: task-success
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/integration-examples 
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: tasks/test_output.yaml
+      params:
+        - name: RESULT
+          value: SUCCESS
+    - name: provision-env-with-ephemeral-namespace
+      taskRef:
+        resolver: bundles
+        params:
+          - name: bundle
+            value: quay.io/redhat-appstudio-tekton-catalog/task-provision-env-with-ephemeral-namespace:0.1-6fa633c8d41e1f885951bfc9d74a5ee9499af79e
+          - name: name
+            value: provision-env-with-ephemeral-namespace
+          - name: kind
+            value: task
+      params:
+        - name: RESULT
+          value: SUCCESS
+        - name: KONFLUXNAMESPACE
+          value: $(context.pipelineRun.namespace)
+        - name: PIPELINERUN_NAME
+          value: $(context.pipelineRun.name)
+        - name: PIPELINERUN_UID
+          value: $(context.pipelineRun.uid)
+    - name: task-success-2
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/integration-examples 
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: tasks/test_output.yaml
+      params:
+        - name: RESULT
+          value: SUCCESS
+    - name: task-skipped
```

Comment:
My vote is to skip this task also to keep the pipeline focused on the important tasks for environment provisioning.